ansible/roles/test/files/ptftests/py3/dhcpv6_counter_test.py

```diff
@@ -137,10 +144,12 @@ def create_unknown_server_packet(self):
     """
 
     def client_send(self):
-        client_messages = [DHCP6_Solicit, DHCP6_Request, DHCP6_Confirm, DHCP6_Renew, DHCP6_Rebind, DHCP6_Release, DHCP6_Decline, DHCP6_Reconf, DHCP6_InfoRequest]
+        client_messages = [DHCP6_Solicit, DHCP6_Request, DHCP6_Confirm, DHCP6_Renew,
+                           DHCP6_Rebind, DHCP6_Release, DHCP6_Decline, DHCP6_InfoRequest]
         for message in client_messages:
             packet = self.create_packet(message)
             testutils.send_packet(self, self.client_port_index, packet)
+            time.sleep(1)
```

Comment:
Can you add a comment here for why one second is needed?

Reply:
In order to avoid single tor issue I mentioned in description. Sometime multicast packets flooded to all ports and make filter socket RECV socket full and finally cause packet drop. So add 1 second here is a work around for this kind of cases (to low down the packet send rate)

Reply:
Applied in latest diff.